internal/filter/filter.go

```diff
@@ -107,6 +107,28 @@ func (f *Filter) FindModulesForNode(node client.Object) []reconcile.Request {
 	return reqs
 }
 
+func (f *Filter) FindPreflightsForModule(mod client.Object) []reconcile.Request {
+	reqs := make([]reconcile.Request, 0)
+
+	logger := f.logger.WithValues("module", mod.GetName())
+	logger.Info("Listing all preflights")
+	preflights := kmmv1beta1.PreflightValidationList{}
+	if err := f.client.List(context.Background(), &preflights); err != nil {
+		logger.Error(err, "could not list preflights")
+		return reqs
+	}
+
+	for _, preflight := range preflights.Items {
+		// skip the preflight being deleted
+		if preflight.GetDeletionTimestamp() != nil {
```

Comment:
I think we already make that check in the reconciler code?

Reply:
no, we check there the modules' deletion condition, not the preflights'